.github/workflows/checks.yml

```diff
@@ -133,7 +133,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: denoland/setup-deno@9db7f66e8e16b5699a514448ce994936c63f0d54 # v1.1.0
         with:
-          deno-version: v1.x
+          deno-version: v1.32.x
```

Comment:
I think we only need `1.32.x` only for the examples.
For the rest, we could leave `1.x`

Reply:
Lint as well since the deno version on our machines will mismatch the CI